run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
nit: how about a makefile?

Reply:
Yeah good idea, at first there was more in this run.sh but as I moved all the vars to using the native flag ENV_VARS, then this file is no longer needed. [897ab4d](https://github.com/Layr-Labs/avs-sync/pull/6/commits/897ab4d67ea0ef343a3048b16770e8bbde6fbf32)